### PR TITLE
feat: display log when when creating a subsystem

### LINKED ISSUE #18: Display log when image is pulled when creating a subsystem

### PR DESCRIPTION
**Pull request by coffandro**
Fixes #18

### CODE REVIEW

**Comment by coffandro** [comment]
Current version "works" with the terminal output but it dosen't yet implement callback, I'm currently switching to Vte.pty instead of Vte.terminal to be able to accomodate a callback function

File: apx_gui/core/apx_entities.py [hunk 1]
```diff
@@ -179,16 +183,24 @@ def __init__(
...
+            f"{self.stack.name}",
         )
         # the following apx command is safe to ignore errors, we´ll check the
         # subsystem status by getting the list of subsystems
-        new_res: tuple[bool, str] = self._run_apx_command(new_command, True)
-        if not new_res[0]:
-            return new_res[0], self
+        self.run_vte_command(new_command, _terminal, self._Create_Callback)
 
+    def _Create_Callback(self,*args):
```
Comment (mirkobrombin) [inline apx_gui/core/apx_entities.py:203]: I would keep this lowercased

File: apx_gui/gtk/create-subsystem.ui [hunk 2]
```diff
@@ -80,10 +80,49 @@
                                         <property name="valign">center</property>
                                     </object>
                                 </child>
+                                <child>
+			                              <object class="GtkButton" id="console_button">
```
Comment (mirkobrombin) [inline apx_gui/gtk/create-subsystem.ui:84]: I think there is an indention issue in the whole block

File: apx_gui/windows/create_subsystem.py [hunk 3]
```diff
@@ -41,6 +41,9 @@ class CreateSubsystemWindow(Adw.Window):
     row_stack: Adw.ComboRow = Gtk.Template.Child()  # pyright: ignore
     str_stack: Gtk.StringList = Gtk.Template.Child()  # pyright: ignore
     stack_main: Adw.ViewStack = Gtk.Template.Child()  # pyright: ignore
+    console_button: Gtk.Box = Gtk.Template.Child()
```
Comment (mirkobrombin) [inline apx_gui/windows/create_subsystem.py:44]: please add a `# pyright: ignore` at the end of each line

**Review by mirkobrombin — CHANGES_REQUESTED** [review]

File: apx_gui/core/apx_entities.py (hunk 1 above)
Comment (coffandro) [reply apx_gui/core/apx_entities.py:203]: Alrighty, I normally use the VariableName style but  I'll do that!

**Comment by coffandro** [comment]
That should fix the issues

**Review by mirkobrombin — APPROVED** [review]
Looks good to me. Can you squash commits into a single one like our guidelines [suggests](https://github.com/Vanilla-OS/.github/blob/main/CONTRIBUTING.md)?

> If deemed necessary, commits should be squashed in the branch directly through git, never through GitHub's "Squash and merge", as that leaves a merge commit behind.

**Comment by coffandro** [comment]
After half an hour of fighting git tooth and nail it is done

**Review by kbdharun — APPROVED** [review]

**Pull request merged**